code/modules/mob/living/simple_animal/hostile/ordeal/salmon.dm

```diff
@@ -0,0 +1,196 @@
+//April Fools Ordeal
```

Comment:
We have code for these to only activate on holidays. Let's get it done so we can full merge this!